jwplayer.license.notice.js

```diff
@@ -4,6 +4,7 @@ const YEAR = new Date().getFullYear();
 const versions = {
     backbone: '1.1.2',
     'can-autoplay': '3.0.0',
+    'focus-options-polyfill': '1.5.0',
```

Comment:
If this package was imported via yarn/npm then we could do
```suggestion
    'focus-options-polyfill': devDeps['focus-options-polyfill'],
```